man/machinery-show.1.md

```diff
@@ -48,9 +48,15 @@ in local time are shown in the title of each scope section.
   * `-p PORT`, `--port=PORT` (optional):
     Specify the port on which the web server will serve the system description: Default: 7585
 
+    Ports can be selected in a range between 1-65535. Ports between 1 and 1023 can only be
+    chosen when `machinery` will be executed as `root` user.
+
   * `-i IP`, `--ip=IP` (optional):
     Specify the IP address on which the web server will be made available. Default: 127.0.0.1
 
+    It's only possible to use an IP address (or hostnames resolving to an IP address) which
+    are assigned to a network interface on the local machine.
```

Comment:
There is a typo here (and all the other copies of the text). It either has to be "an IP address (...) which is assigned" or "IP addresses (...) which are assigned".